optimism/src/keccak/witness.rs

```diff
@@ -257,7 +256,15 @@ impl<Fp: Field> KeccakInterpreter for KeccakEnv<Fp> {
         chi.state_f()
     }
 
-    fn run_iota(&mut self, _state_f: &[u64], _rc: &[u64]) {
-        todo!()
+    fn run_iota(&mut self, state_f: &[u64], round: usize) {
+        let iota = Iota::create(state_f, round);
+
+        // Update columns
+        for i in 0..QUARTERS * DIM * DIM {
+            self.write_column(KeccakColumn::NextState(i), iota.state_g(i));
+        }
+        for i in 0..QUARTERS {
+            self.write_column(KeccakColumn::RoundConstants(i), iota.rc(i));
```

Comment:
I think we do not need columns for the round constants. It is well-defined by the Keccak instance, and static.

Reply:
And [this](https://github.com/o1-labs/proof-systems/pull/1558/files#r1430167003) is related.

Reply:
The goal for having these is to be able to check in a lookup table that the right round constant is being used in the current round. 

But if it makes more sense to model it differently I will be glad to adapt it.

Reply:
>The goal for having these is to be able to check in a lookup table that the right round constant is being used in the current round.

Oh, I see. Never thought about this. I do not know if it is important.

Reply:
Opened https://github.com/o1-labs/proof-systems/issues/1608 to keep track of it.